src/components/Header/Header.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Header } from './Header.component';
+
+describe('Header', () => {
+  beforeEach(() => {
+    render(<Header />);
+  });
+
+  test('should contains a search field', () => {
+    const searchField = screen.queryByPlaceholderText(/search/i);
```

Comment:
It is better to not use `queryBy` selectors when testing for things to be in the document

[source](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence)

Reply:
That explains all the errors in my coverage xD